node/node.go

```diff
@@ -267,35 +268,48 @@ func createAndStartIndexerService(
 	dbProvider DBProvider,
 	eventBus *types.EventBus,
 	logger log.Logger,
-) (*indexer.Service, indexer.TxIndexer, indexer.BlockIndexer, error) {
-
-	var (
-		txIndexer    indexer.TxIndexer
-		blockIndexer indexer.BlockIndexer
-	)
+) (*indexer.Service, []indexer.EventSink, error) {
+
+	eventSinks := []indexer.EventSink{}
+	var sqlDB *sql.DB
+loop:
+	for _, db := range config.TxIndex.Indexer {
+		switch strings.ToLower(db) {
+		case string(indexer.NULL):
+			// when we see null in the config, the eventsinks will be reset with the nullEventSink.
+			eventSinks = append([]indexer.EventSink{}, nullSink.NewNullEventSink())
```

Comment:
We need to ensure when a `null` sink is set, that we _only_ have that sink 👍 

```suggestion
			eventSinks = []indexer.EventSink{nullSink.NewNullEventSink()}
```